core/src/main/java/com/flipkart/gjex/core/web/RotationManagementResource.java

```diff
@@ -0,0 +1,47 @@
+package com.flipkart.gjex.core.web;
+
+import com.flipkart.gjex.core.healthcheck.RotationManagementBasedHealthCheck;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * Resource for rotation management
+ * @author ajay.jalgaonkar
+ */
+
+@Singleton
+@Path("/")
+@Named
+public class RotationManagementResource {
+
+  @Inject
+  public RotationManagementResource(RotationManagementBasedHealthCheck rotationManagementBasedHealthCheck) {
+    this.rotationManagementBasedHealthCheck = rotationManagementBasedHealthCheck;
+  }
+
+  private RotationManagementBasedHealthCheck rotationManagementBasedHealthCheck;
+
+  @GET
```

Comment:
@ajay-jalgaonkar Lets not have this as GET's. A modification status should be an request which cannot be triggered by mistake or by automatic refresh. Lets move this to POST. Same with bir.

Reply:
done